contracts/src/v0.8/keeper2_0/KeeperRegistry2_0.sol

```diff
@@ -47,13 +50,18 @@ contract KeeperRegistry2_0 is
    * - KeeperRegistry 1.1.0: added flatFeeMicroLink
    * - KeeperRegistry 1.0.0: initial release
    */
-  string public constant override typeAndVersion = "KeeperRegistry 2.0.0";
+  string public constant override typeAndVersion = "KeeperRegistry 2.1.0";
```

Comment:
2.0.1? Not sure, we have been calling next version 2.1 so would reduce confusion from that

Reply:
I guess it does feel like more of a patch anyway - will change